2020/day3.clj

```diff
@@ -1,89 +0,0 @@
-(ns advent-of-code.2020.day3
-  (:require [clojure.string :as string]))
-
-(defn file->seq
-  [filename]
-  (->> (slurp filename)
-       (string/split-lines)))
-
-(defn next-location
-  [loc [right down]]
```

Comment:
Why loc? first and second are not specific -- re-read 'names' in Elements of Clojure

Reply:
> Why loc? first and second are not specific -- re-read 'names' in Elements of Clojure

Solved it in second PR.